crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -541,6 +541,27 @@ en:
       active: 'Active'
       active_repo_tooltip: 'Managed nodes will be automatically configured to use repositories only if they are marked as active'
 
+  installer:
+    status:
+      installation_failed: 'Installation failed prematurely. Please examine /var/log/crowbar/install.log'
```

Comment:
Can it fail non-prematurely? ;-) Just drop that word. And maybe s/examine/check/